openedx/features/classrooms/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class ClassroomsConfig(AppConfig):
+    name = 'classrooms'
```

Comment:
Please have a look at this: https://docs.djangoproject.com/en/3.0/ref/applications/#django.apps.AppConfig.name

Reply:
Corrected it.